enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.unit.spec.tsx

```diff
@@ -67,13 +73,16 @@ const setup = ({
 
   setupCardQueryEndpoints(TEST_CARD, TEST_DATASET);
 
-  renderWithProviders(<InteractiveQuestion questionId={TEST_CARD.id} />, {
-    mode: "sdk",
-    sdkConfig: createMockConfig({
-      jwtProviderUri: "http://TEST_URI/sso/metabase",
-    }),
-    storeInitialState: state,
-  });
+  return renderWithProviders(
```

Comment:
return the result from `renderWithProvider` so I could access `store` and use it to dispatch actions.